src/Functions/Az.Functions.psd1

```diff
@@ -1,7 +1,7 @@
 @{
   GUID = 'eafced71-8742-4a2c-5afd-13117428dd90'
   RootModule = './Az.Functions.psm1'
-  ModuleVersion = '1.0.0'
+  ModuleVersion = '1.0.1'
```

Comment:
Module version doesn't need to be changed.

Reply:
@dingmeng-xue should I revert this file? Or keep it as is? 

Please note that I have also updated `src/Functions/readme.md` to increase the module version to `1.0.1`.

Reply:
@dingmeng-xue reverted back `src/Functions/Az.Functions.psd1`